src/routers/project/projects.py

```diff
@@ -0,0 +1,73 @@
+from fastapi import APIRouter, HTTPException, Response
+
+from src.project_handler_factory import ProjectHandlerType, createHandler
+from src.routers.project.schemas import NewProject, ProjectData
+
+router = APIRouter()
+projectHandler = createHandler(ProjectHandlerType.IN_MEMORY)

```

Comment:
Such functionality that is reused can be passed as a dependency to routers
https://fastapi.tiangolo.com/tutorial/dependencies/

Reply:
I read about dependencies and from what I understood about them I planned to use them when I implement authentication logic with JWTs. 
However, in this case of project_handler, I don't really understand how to make it a dependency. project_handler is supposed to be one instance (in case of InMemProjectHandler must always the same instance) of the ProjectHandler class that is available to all API endpoints. Can you explain to me a bit more how I can turn it into a dependency and what added value it brings compared to it just being a global variable?

Reply:
Included your suggestions in later iterations of code, so will close this now.